test/bin/manage_hypervisor_config.sh

```diff
@@ -159,6 +169,13 @@ action_cleanup() {
 
     # Stop nginx web server
     "${TESTDIR}/bin/manage_webserver.sh" "stop"
+
+    if [ -e "${VGFILE}" ]; then
```

Comment:
As above, this might need to be part of `scenario.sh` script.

Reply:
Not really. VG is something shared across the containers, so I didn't want to `scenario.sh cleanup` to remove a VG that might be used by different container.
I'm curious though if we could run these commands inside a container to get isolated VGs.

Reply:
I'm concerned how this is going to work if we have multiple containers accessing the same vg.
Apparently, there is --device option in podman that allows mapping only specific devices.
Can you try to see if this addresses our concerns?